roles/bootstrap-os/tasks/main.yml

```diff
@@ -73,3 +78,6 @@
     filter: ansible_hostname
   when:
     - hostname_changed.changed
+
+- name: Install httplib2
```

Comment:
Installation of httplib2 is done via system packages for all but coreos distribs so this task should be removed from here and added to bootstrap-oracle.yml instead (in the same way as it is done for coreos maybe)

Reply:
Changed. Thanks

Reply:
This is an outdated diff. This has already been changed.